src/main/java/uk/gov/hmcts/darts/audio/entity/MediaRequestEntity.java

```diff
@@ -69,6 +69,9 @@ public class MediaRequestEntity extends CreatedModifiedBaseEntity {
     @Enumerated(EnumType.STRING)
     private AudioRequestType requestType;
 
+    @Column(name = CREATED_TS_COLUMN_NAME, nullable = false)
+    private OffsetDateTime createdDateTime;
```

Comment:
!! Adding this column here doesn't look right to me as we already have it in the extends class

Reply:
Well spotted @richardbull, removed here https://github.com/hmcts/darts-api/pull/485